keycloak_setup/institution_list.py

```diff
@@ -28,371 +30,619 @@ class InstitutionMeta(TypedDict):
         "region": _US_CAN,
+        "authorlist": True,
+        "_ldap_o": "uaa",
     },
     "Alberta": {
         "cite": "Dept. of Physics, University of Alberta, Edmonton, Alberta, Canada T6G 2E1",
         "abbreviation": "ALBERTA",
         "is_US": False,
         "region": _US_CAN,
+        "authorlist": True,
+        "_ldap_o": "alberta",
+    },
+    "Barbados": {
+        "cite": "Dept. of Physics, University of the West Indies, Cave Hill Campus, Bridgetown BB11000, Barbados",
+        "abbreviation": "BARBADOS",
+        "is_US": False,
+        "region": _US_CAN,
+        "authorlist": False,
+        "_ldap_o": "uwi",
     },
     "Berlin": {
         "cite": "Institut für Physik, Humboldt-Universität zu Berlin, D-12489 Berlin, Germany",
         "abbreviation": "HUMBOLDT",
         "is_US": False,
         "region": _EUROPE,
+        "authorlist": True,
+        "_ldap_o": "humboldt",
     },
     "Bochum": {
         "cite": "Fakultät für Physik & Astronomie, Ruhr-Universität Bochum, D-44780 Bochum, Germany",
         "abbreviation": "BOCHUM",
         "is_US": False,
         "region": _EUROPE,
+        "authorlist": True,
+        "_ldap_o": "rub",
+    },
+    "Bonn": {
+        "cite": "hysikalisches Institut, Universität Bonn, Nussallee 12, D-53115 Bonn, Germany",
+        "abbreviation": "BONN",
+        "is_US": False,
+        "region": _EUROPE,
+        "authorlist": False,
+        "_ldap_o": "bonn",
     },
     "Brussels-ULB": {
         "cite": "Université Libre de Bruxelles, Science Faculty CP230, B-1050 Brussels, Belgium",
         "abbreviation": "ULB",
         "is_US": False,
         "region": _EUROPE,
+        "authorlist": True,
+        "_ldap_o": "ulb",
     },
     "Brussels-VUB": {
         "cite": "Vrije Universiteit Brussel (VUB), Dienst ELEM, B-1050 Brussels, Belgium",
         "abbreviation": "VUB",
         "is_US": False,
         "region": _EUROPE,
+        "authorlist": True,
+        "_ldap_o": "vub",
     },
     "Canterbury": {
         "cite": "Dept. of Physics and Astronomy, University of Canterbury, Private Bag 4800, Christchurch, New Zealand",
         "abbreviation": "UC",
         "is_US": False,
         "region": _ASIA_PAC,
+        "authorlist": True,
+        "_ldap_o": "canterbury",
     },
     "Chiba": {
         "cite": "Dept. of Physics and Institute for Global Prominent Research, Chiba University, Chiba 263-8522, Japan",
         "abbreviation": "CHIBA",
         "is_US": False,
         "region": _ASIA_PAC,
+        "authorlist": True,
+        "_ldap_o": "chiba",
     },
     "Clark-Atlanta": {
         "cite": "CTSPS, Clark-Atlanta University, Atlanta, GA 30314, USA",
         "abbreviation": "CAU",
         "is_US": True,
         "region": _US_CAN,
-    },
-    "Columbia": {
-        "cite": "Columbia Astrophysics and Nevis Laboratories, Columbia University, New York, NY 10027, USA",
-        "abbreviation": "COLUMBIA",
-        "is_US": True,
-        "region": _US_CAN,
```

Comment:
Why is Columbia being removed?

Reply:
It's a Gen2 institution in the authorlist, not regular IceCube.  You can find it below.

Reply:
_marking as resolved since this is the same as below_